Plugins/Gerrit/Server/GerritCapabilities.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using ResourceManager;
+
+namespace Gerrit.Server
+{
+    public class GerritCapabilities : Translate
+    {
+        #region Translation
+        private static readonly TranslationString _publishTypeReview = new TranslationString("For Review");
+        private static readonly TranslationString _publishTypeWip = new TranslationString("Work-in-Progress");
+        private static readonly TranslationString _publishTypePrivate = new TranslationString("Private");
+        private static readonly TranslationString _publishTypeDraft = new TranslationString("Draft");
+        #endregion
```

Comment:
I have moved these from FormGerritPublish.cs but UpdateLocalEnglishTranslations script did not add them to the xlf file.
How to handle this?